diff --git a/node_modules/react-native-zip-archive/RNZipArchive.podspec b/node_modules/react-native-zip-archive/RNZipArchive.podspec
index d61fbd3..a339512 100644
--- a/node_modules/react-native-zip-archive/RNZipArchive.podspec
+++ b/node_modules/react-native-zip-archive/RNZipArchive.podspec
@@ -10,11 +10,11 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.homepage       = 'https://github.com/mockingbot/react-native-zip-archive'
   s.source         = { :git => 'https://github.com/mockingbot/react-native-zip-archive.git', :tag => "#{s.version}"}
-  s.platform       = :ios, '15.5'
+  s.platform       = :ios, '9.0'
   s.preserve_paths = '*.js'
 
   s.dependency 'React-Core'
-  s.dependency 'SSZipArchive', '~>2.5.5'
+  s.dependency 'SSZipArchive', '2.4.3'
 
   s.subspec 'Core' do |ss|
     ss.source_files = 'ios/*.{h,m}'
