diff --git a/node_modules/react-native-view-shot/ios/RNViewShot.m b/node_modules/react-native-view-shot/ios/RNViewShot.m
index 9015847..670e1bf 100644
--- a/node_modules/react-native-view-shot/ios/RNViewShot.m
+++ b/node_modules/react-native-view-shot/ios/RNViewShot.m
@@ -54,8 +54,10 @@ - (dispatch_queue_t)methodQueue
     if ([target intValue] == -1) {
       UIWindow *window = [[UIApplication sharedApplication] keyWindow];
       view = window;
-    } else {
+    } else if (viewRegistry[target] != nil) {
       view = viewRegistry[target];
+    } else {
+      view = [uiManager viewForReactTag:target];
     }
 
     if (!view) {
