diff --git a/node_modules/react-native-slider/lib/Slider.js b/node_modules/react-native-slider/lib/Slider.js
index c640410..b8f7d50 100644
--- a/node_modules/react-native-slider/lib/Slider.js
+++ b/node_modules/react-native-slider/lib/Slider.js
@@ -5,6 +5,8 @@ var _react=require("react");var _react2=_interopRequireDefault(_react);
 
 
 var _reactNative=require("react-native");
+var _deprecatedReactNativePropTypes = require("deprecated-react-native-prop-types");
+
 
 
 
@@ -549,11 +551,11 @@ return false;}}]);return Slider;}(_react.PureComponent);Slider.propTypes={ /**
      * the slider is released).
      */onSlidingComplete:_propTypes2.default.func, /**
      * The style applied to the slider container.
-     */style:_reactNative.ViewPropTypes.style, /**
+     */style:_deprecatedReactNativePropTypes.style, /**
      * The style applied to the track.
-     */trackStyle:_reactNative.ViewPropTypes.style, /**
+     */trackStyle:_deprecatedReactNativePropTypes.style, /**
      * The style applied to the thumb.
-     */thumbStyle:_reactNative.ViewPropTypes.style, /**
+     */thumbStyle:_deprecatedReactNativePropTypes.style, /**
      * Sets an image for the thumb.
      */thumbImage:_reactNative.Image.propTypes.source, /**
      * Set this to true to visually see the thumb touch rect in green.
diff --git a/node_modules/react-native-slider/src/Slider.js b/node_modules/react-native-slider/src/Slider.js
index 37deee5..5912aef 100644
--- a/node_modules/react-native-slider/src/Slider.js
+++ b/node_modules/react-native-slider/src/Slider.js
@@ -11,9 +11,9 @@ import {
   PanResponder,
   View,
   Easing,
-  ViewPropTypes
 } from "react-native";
 
+import ViewPropTypes from 'deprecated-react-native-prop-types';
 import PropTypes from 'prop-types';
 
 var TRACK_SIZE = 4;
